docs/pages/repo/docs/reference/configuration.mdx

```diff
@@ -57,6 +57,12 @@ A list of environment variables for implicit global hash dependencies. The conte
 }
 ```
 
+## `extends`
+
+`type: string[]`
+
+The `extends` key is only valid in [Workspace Configuration][1]. It will be ignored in the root `turbo.json`.
+
```

Comment:
Maybe a "To learn more, <link>" to get the full rundown if I landed here from search and still feel lost?